docs/template_plugin/src/CMakeLists.txt

```diff
@@ -23,11 +23,18 @@ target_include_directories(${TARGET_NAME} PRIVATE
     "${CMAKE_CURRENT_SOURCE_DIR}"
     "${IE_MAIN_TEMPLATE_PLUGIN_SOURCE_DIR}/include")
 
-target_link_libraries(${TARGET_NAME} PRIVATE IE::inference_engine IE::inference_engine_transformations ${NGRAPH_LIBRARIES} ${INTEL_ITT_LIBS})
+target_link_libraries(${TARGET_NAME} PRIVATE
+    IE::inference_engine
+    IE::inference_engine_transformations
+    ${INTEL_ITT_LIBS}
+    ${NGRAPH_LIBRARIES}
+    ngraph_backend
+    interpreter_backend
+    half)
 
 # ATTENTION: uncomment to register a plugin in the plugins.xml file
-# ie_register_plugins(MAIN_TARGET ${TARGET_NAME}
-                    # POSSIBLE_PLUGINS ${TARGET_NAME})
+ie_register_plugins(MAIN_TARGET ${TARGET_NAME}
```

Comment:
Do we remove template_plugin from `plugins.xml` when prepare a OpenVINO package?